Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -779,8 +779,7 @@ WebIDL::ExceptionOr<void> XMLHttpRequest::send(Optional<DocumentOrXMLHttpRequest
 
             // 2. Handle errors for this.
             // NOTE: This cannot throw, as `handle_errors` only throws in a synchronous context.
-            // FIXME: However, we can receive allocation failures, but we can't propagate them anywhere currently.
-            handle_errors().release_value_but_fixme_should_propagate_errors();
+            MUST(handle_errors());
```

Comment:
I believe this is correct, the only place this can throw is in `request_errors_steps()`, and then only when we are in a synchronous context, same goes for `handle_response_end_of_body()`

https://github.com/LadybirdBrowser/ladybird/blob/cb1a1a5cb5838489472c0135c17b57c1ef3c2b2c/Libraries/LibWeb/XHR/XMLHttpRequest.cpp#L1251-L1254